lib/game.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class Game
+  attr_accessor :cell
+
+  def initialize(cell)
+    @cell = cell
+  end
+
+  def win_game
+    diagonal || vertical || horizontal
+  end
+
+  private
+
+  def diagonal
+    left_diagonal || right_diagonal
+  end
+
+  def left_diagonal
+    @cell[0][0] == @cell[1][1] && @cell[1][1] == @cell[2][2]
+  end
+
+  def right_diagonal
+    @cell[2][0] == @cell[1][1] && @cell[1][1] == @cell[0][2]
+  end
+
+  def vertical
+    vertical_one || vertical_two || vertical_three
+  end
+
+  def vertical_one
+    @cell[0][0] == @cell[1][0] && @cell[1][0] == @cell[2][0]
+  end
+
+  def vertical_two
+    @cell[0][1] == @cell[1][1] && @cell[1][1] == @cell[2][1]
+  end
+
+  def vertical_three
+    @cell[0][2] == @cell[1][2] && @cell[1][2] == @cell[2][2]
+  end
+
+  def horizontal
+    horizontal_one || horizontal_two || horizontal_three
+  end
+
+  def horizontal_one
+    @cell[0][0] == @cell[0][1] && @cell[0][1] == @cell[0][2]
+  end
+
+  def horizontal_two
+    @cell[1][0] == @cell[1][1] && @cell[1][1] == @cell[1][2]
+  end
```

Comment:
[OPTIONAL]
Can you think of a more optimal way to check this?
Currently, you are checking each possibility once. 
Consider if there may be a simpler way.

Reply:
As @mcrd25 said this code is to hardcoded. 
You can use an approach like this

  def horizontal
    hor = [
      [@cell[0][0], @cell[0][1], @cell[0][2]],
      [@cell[1][0], @cell[1][1], @cell[1][2]],
      [@cell[2][0], @cell[2][1], @cell[2][1]]
    ]
  end

```
def check_win?(array)
    array.each {|cell| cell.all?('X') }
end
```
This is untested but its just to give an example of how you can approach this.